test/tests/dgkernels/ad_dg_diffusion/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
+  [./test]
+    type = 'Exodiff'
+    input = '2d_diffusion_ad_dg_test.i'
+    exodiff = '2d_diffusion_ad_dg_test_out.e-s003'
```

Comment:
Why the `-s003` extension? We're not doing adaptivity right? Seems like we only need the `2d_diffusion_ad_dg_test_out.e` binary...?

Reply:
Ah, I see the `Adaptivity` block now. let's test both files